airflow/ui/src/pages/DagsList/DagsList.tsx

```diff
@@ -200,21 +200,33 @@ export const DagsList = () => {
           <Heading py={3} size="md">
             {pluralize("DAG", data?.total_entries)}
           </Heading>
-          {display === "card" ? (
-            <Select
-              data-testid="sort-by-select"
-              onChange={handleSortChange}
-              placeholder="Sort by…"
-              value={orderBy}
-              variant="flushed"
-              width="200px"
-            >
-              <option value="dag_id">Sort by DAG ID (A-Z)</option>
-              <option value="-dag_id">Sort by DAG ID (Z-A)</option>
-            </Select>
-          ) : (
-            false
-          )}
+          <Select
+            data-testid="sort-by-select"
+            onChange={handleSortChange}
+            placeholder="Sort by…"
+            value={orderBy}
+            variant="flushed"
+            width="200px"
+          >
+            <option value="dag_display_name">Sort by Display Name (A-Z)</option>
```

Comment:
I don't thing we shoult remove the `dag_id` option

Reply:
In discussion with @bbovenzi , 
The thought behind removing the `dag id` is that, `dag_display_name` has a fall back search for `dag id`.
But I also think since user expectation would be that they are sorting based upon `dag_display_name` and since it's not a mandatory field, it would have many null values, and probably it would end with sorting some field with `dag_display_name` and other with `dag id`. Maybe we should have this info somewhere (hover) ?

Reply:
I definitely want to move the new UI to only use `dag_display_name`. dag_id is only a detail the user needs for debugging. But yes, we need to update our sorting in FastAPI to fallback to dag_id just like we do for search query.

Reply:
Ok got it. Yes normally default sorting id `dag_id` so we should be able to just remove it.